.github/workflows/jre8-release.yml

```diff
@@ -38,5 +38,8 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.CR_PAT }}
 
-      - name: Push Docker image
-        run: docker push ghcr.io/scalar-labs/jre8:${{ steps.version.outputs.version }}
+      - name: Rebuild image for multiple platforms and push Docker image
+        run: |
+          docker buildx create --use
```

Comment:
Not a comment but more like a question: there is no need to remove the buildx builder after the build and push are completed? I guess it's all removed anyway.

Reply:
I guess it's ok since the instance will be destroyed.
But, just in case let me remove it
https://github.com/scalar-labs/docker/pull/17/commits/29787dd95605da202cb92d582cb376aeb28f08e5